docs/self-managed/reference-architecture/reference-architecture.md

```diff
@@ -0,0 +1,149 @@
+
+High availability (HA) ensures that a system remains operational and accessible even in the event of component failures. Generally all components are equipped to be run in a highly available manner. Some components do need extra considerations when run in HA mode.
+
+<!-- TODO Describe Optimize and Connectors limitations or point to resource for more -->
+
+While high availability is one part of the increased fault tolerance and resilience, you should also consider regional or zonal placement of your workloads.
+
+If you run infrastructure on cloud providers, you are often met with different regions and zones. For ideal high availability you should consider a minimum setup of 3 zones within a region as this will guarantee that in case of a zonal failure that the remaining two workloads can still process data. For more information on how Zeebe handles fault tolerance, have a look at the [raft consensus chapter](/components/zeebe/technical-concepts/clustering.md#raft-consensus-and-replication-protocol).
+
+If running a single instance is preferred, make sure to implement [regular backups](/self-managed/operational-guides/backup-restore/backup-and-restore.md) since resilience will be limited.
+
+## Camunda 8 Deployment Reference architectures
+
+### Kubernetes
+
+Kubernetes is a powerful orchestration platform for containerized applications. Using a reference architecture for Kubernetes can help organizations deploy and manage their applications more effectively. It provides guidelines for setting up clusters, managing workloads, and ensuring high availability and scalability. This approach is ideal for organizations looking to leverage the benefits of containerization and self-healing capabilities.
+
+<!-- TODO add link or card for AWS ref arch -->
+
+### Containers
+
+Containers, such as Docker, offer a middle ground between the manual JAR and Kubernetes approaches. They provide a lightweight, portable, and consistent runtime environment, making it easier to develop, test, and deploy applications across different environments. Containers encapsulate an application and its dependencies, ensuring that it runs reliably regardless of where it is deployed.
+
+### Manual JAR (Bare Metal / VMs)
+
+For organizations that prefer traditional infrastructure, reference architectures for bare metal or virtual machines (VMs) offer a structured approach to system deployment. These architectures provide best practices for setting up physical servers or VMs, configuring networks, and managing storage using Infrastructure as Service cloud roviders. They are suitable for environments where containerization or use of Kubernetes services may not be feasible.
+
+<!-- TODO add link or card for AWS EC2 -->
+
+### Local Development
+
+While both options are suitable for trying out Camunda 8 locally, you might also consider exploring [Camunda 8 Run](/self-managed/setup/deploy/local/c8run.md) for a developer focused experience.
+
+## Helping Customers Decide
```

Comment:
I think we can move the bullet points for these sections up to the k8s, containers, manual sections above - that way they are combined/the information is more compact and less duplicated. 

If there is a strong desire to separate this information out, I think a separate and more dedicated "Which option is right for me?" guide would be preferable to including it here, and I'd recommend folding the sections together for now.

Reply:
This pulls the links to each individual reference architecture guide up as well, which is nice for this section, too.

Reply:
I'd be happy if you could implement those changes 🙇!